### DISCUSSION TITLE
Formatting data for use in histograms

### DISCUSSION THREAD

**Discussion by Bordocklius** [grafana/grafana-json-datasource#107] [Q&A]
Hi everyone,

I wanted to make a few histograms for some of my data. Although it apears just fine in the dashboard, i read somewhere that you can pre format data into buckets for a histogram. I can't find how i'm supposed to format that, so i thought i would ask it here.

Thanks in advance.

**Comment by marcusolsson** [comment]
The format depends on the panel you're using. Do you know which panel you're hoping to use to display the histogram?

**Reply by Bordocklius** [reply]
![afbeelding](https://user-images.githubusercontent.com/58853886/117933416-d5e7a780-b301-11eb-84a1-b2b0f76d8f04.png)

I'm currently using the regular graph visualisation. In the axis tab, i switched mode to Histogram

![afbeelding](https://user-images.githubusercontent.com/58853886/117933522-f0ba1c00-b301-11eb-97e4-a7baa7c54f47.png)

**Comment by marcusolsson** [comment]
Hm. Pretty sure  that the Graph panel histogram computes the histogram from the time series, but I'll be able to confirm this next week.
